Connector/wsutils/websocket.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python
+from logger import logger
+
+# TODO: Create sync lock over webSockets
```

Comment:
Check this todo and following comments

Reply:
More comments will be added in difficult code sections in the documentation update